low-liquidity-market-attack-monitor/agent-config.json

```diff
@@ -0,0 +1,19 @@
+
+{
+  "developerAbbreviation": "AE",
+  "protocolName": "Compound",
+  "protocolAbbreviation": "COMP",
+  "contracts": {
+    "Comptroller": {
+      "address": "0x3d9819210A31b4961b30EF54bE2aeD79B9c9Cd3B",
+      "abiFile": "Comptroller.json"
+    },
+    "CompoundToken": {
+      "address": "0xc00e94cb662c3520282e6f5717214004a7f26888",
```

Comment:
This does not appear to be used anywhere in a substantive way.  It's also a little confusing to have the main key called `CompoundToken`, but that's not really what this entry is used for in the code (i.e. the COMP token isn't involved in this Bot).

Reply:
Yea it's a bad naming collision... cToken vs Compound Token